invenio_rdm_records/services/schemas/record.py

```diff
@@ -88,3 +90,21 @@ def default_nested(self, data, many, **kwargs):
             data["custom_fields"] = {}
 
         return data
+
+    def hide_tombstone_or_metadata(self, data):
+        """Hide tombstone info if the record isn't deleted and metadata if it is."""
+        if not data.get("deletion_status", {}).get("is_deleted", False):
+            data.pop("tombstone", None)
+        else:
+            data.pop("metadata", None)
+            data.pop("files", None)
+            data.pop("stats", None)
```

Comment:
not sure which fields we should actually hide, these were just a few that came to my mind.
probably we'll also need to adapt the generation of links as well, s.t. we don't generate links to something that won't work anyway (like `files`).

Reply:
Yep....and probably we shoudl remove all the other info from the record besides the tombstone......

Reply:
the current version right now pops most other fields, resulting in:
```python
In [5]: svc.read(system_identity, pid).data
Out[5]:
{'id': 'zm408-gyp73',
 'links': {'self': 'https://127.0.0.1/api/records/zm408-gyp73',
  'self_html': 'https://127.0.0.1/records/zm408-gyp73',
  'parent': 'https://127.0.0.1/api/records/806dg-1x022',
  'parent_html': 'https://127.0.0.1/records/806dg-1x022',
  'self_iiif_manifest': 'https://127.0.0.1/api/iiif/record:zm408-gyp73/manifest',
  'self_iiif_sequence': 'https://127.0.0.1/api/iiif/record:zm408-gyp73/sequence/default',
  'files': 'https://127.0.0.1/api/records/zm408-gyp73/files',
  'media_files': 'https://127.0.0.1/api/records/zm408-gyp73/media-files',
  'archive': 'https://127.0.0.1/api/records/zm408-gyp73/files-archive',
  'latest': 'https://127.0.0.1/api/records/zm408-gyp73/versions/latest',
  'latest_html': 'https://127.0.0.1/records/zm408-gyp73/latest',
  'draft': 'https://127.0.0.1/api/records/zm408-gyp73/draft',
  'versions': 'https://127.0.0.1/api/records/zm408-gyp73/versions',
  'access_links': 'https://127.0.0.1/api/records/zm408-gyp73/access/links',
  'reserve_doi': 'https://127.0.0.1/api/records/zm408-gyp73/draft/pids/doi',
  'communities': 'https://127.0.0.1/api/records/zm408-gyp73/communities',
  'communities-suggestions': 'https://127.0.0.1/api/records/zm408-gyp73/communities-suggestions',
  'requests': 'https://127.0.0.1/api/records/zm408-gyp73/requests'},
 'pids': {'oai': {'identifier': 'oai:my-site.com:zm408-gyp73',
   'provider': 'oai'}},
 'tombstone': {'removal_reason': {'id': 'spam',
   'title': {'de': 'Spam', 'en': 'Spam'}},
  'note': '',
  'removed_by': {'user': 'system'},
  'removal_date': '2023-08-04',
  'citation_text': '',
  'is_visible': True,
  'identifiers': [{'scheme': 'oai',
    'identifier': {'identifier': 'oai:my-site.com:zm408-gyp73',
     'provider': 'oai'}}]},
 'deletion_status': {'is_deleted': True, 'status': 'D'}}
```

note that `tombstone.identifiers` is effectively just a massaged clone of `pids`.